pkg/api/blackduck/v1/types.go

```diff
@@ -50,26 +50,26 @@ func (b *Blackduck) GenPVC(defaultPVC map[string]string) ([]*components.Persiste
 			pvcMap[claim.Name] = claim
 		}
 
-		for name, defaultSize := range defaultPVC {
```

Comment:
we should move the method out of v1/types.go. It's not mandatory.

Reply:
This seems relevant to the BD type, but probably shouldn't be in the api structure.  I'm find moving it if it makes sense, but in a separate PR.